package com.abstracta.opencart.automation.exceptions;

public class RegistroFallidoExceptions extends AssertionError{

    public RegistroFallidoExceptions(String message, Throwable cause) {
        super(message, cause);
    }
}

